source/Nevermore/Advanced/RelationalTransactionRegistry.cs

```diff
@@ -12,14 +12,21 @@
 
 namespace Nevermore.Advanced
 {
-    public class RelationalTransactionRegistry
+    public interface IRelationalTransactionRegistry
+    {
+        void Add(ReadTransaction trn);
```

Comment:
Agree that it could be improved, I chose to keep that out of scope though. FWIW the reason I added the interface is to provide a no-op implementation so we can implement a better alternative completely outside of Nevermore.

This class is a bit odd already, it logs a message with `The below information may help the Octopus team diagnose the problem` which suggests it shouldn't be in Nevermore to begin with.